src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.PackageDependencyResolution.targets

```diff
@@ -397,15 +397,14 @@ Copyright (c) .NET Foundation. All rights reserved.
 
       <Output TaskParameter="CopyLocalItems" ItemName="_ContentCopyLocalItems" />
       <Output TaskParameter="ProcessedContentItems" ItemName="_ProcessedContentItems" />
-      <Output TaskParameter="FileWrites" ItemName="FileWrites" />
+      <Output TaskParameter="ProcessedContentItems" ItemName="FileWrites" />
```

Comment:
I'm not following what this chance achieves, maybe someone more familiar with this task could take a look? @dsplaisted does this change look okay to you?

Reply:
The itemgroup "FileWrites" contains a list of items that the this project has or would produce.  The task output "FileWrites" is a list of files that has copied.  On incremental build, this task copied no files, thus "FileWrites" itemgroup receives less files.  At the end of the build, incremental clean step will delete files not in itemgroup "FileWrites".  This delete the files copied by this task.  The next build will re-copy the files and the cycle repeats.

Incremental build would break every other build.

Reply:
I think this change makes sense.  Can we delete the `FileWrites` output parameter from the `ProduceContentAssets` task, since it's not used anymore and if it is used it's probably going to cause incremental build bugs.

Reply:
If I was main owner, I would agree.  My change is focused on low risk and low chance of regression.  I don't want to trackdown everyone else that uses it, even though it's probably zero. Feel free to take over.